src/pages/CreatePool/index.tsx

```diff
@@ -110,34 +142,58 @@ export default function CreatePool(): React.ReactElement {
     }
   }
 
-  const poolNameError = poolName.length > 10
-  const poolSymbolError = poolSymbol.length > 14
-  const parameterError =
-    !isNumber(parameter) ||
-    parseFloat(parameter) < 1 ||
-    parseFloat(parameter) > 400
+  const poolNameError = poolName.length > 31
+  const poolSymbolError = poolSymbol.length > 31
+  const aParameterError = !isNumber(aParameter) || parseFloat(aParameter) < 1
 
   const feeError =
     !isNumber(fee) || parseFloat(fee) > 1 || parseFloat(fee) < 0.04
 
   useEffect(() => {
-    const ishavingErrorField =
-      poolNameError || poolSymbolError || parameterError || feeError
-    const ishavingAllValue =
+    const getBasePoolLPTokenAddrs = async () => {
+      if (poolRegistry) {
+        const basePoolName = poolType === "usdMetapool" ? "USDv2" : "BTCv2"
+        try {
+          const poolRegistryData = await poolRegistry.getPoolDataByName(
+            ethers.utils.formatBytes32String(basePoolName),
+          )
+          setMetapoolBasepoolAddr(poolRegistryData.poolAddress)
+          setMetapoolBasepoolLpAddr(poolRegistryData.lpToken)
+        } catch (err) {
```

Comment:
if the registry call fails, should any state be cleared or errors raised?

Reply:
If it fails then the state will be an empty string. Users aren't going to be able to create a permissionless pool without this and there's no user input for this value.